docs/cinder-csi-plugin/development.md

```diff
@@ -0,0 +1,16 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Development tips for Cinder CSI](#development-tips-for-cinder-csi)
+  - [Update CSI spec version and Cinder CSI driver version](#update-csi-spec-version-and-cinder-csi-driver-version)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+
+# Development tips for Cinder CSI
+
+## Update CSI spec version and Cinder CSI driver version
+
+There's two versions (`specVersion` and `Version`) defined at [driver.go](https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/driver.go) and both of them are in `x.y.z` format. `specVersion` which indicate Cinder CSI followed which version of [CSI spec](https://github.com/container-storage-interface/spec) and `Version` is the version of Cinder CSI driver itself. For each CSI driver functionalities update such as options/params updated, you need increase `.z` version. If the CSI spec version is upgraded, the Cinder CSI version need bump as well.
```

Comment:
s/There's/There are

Reply:
`specVersion` indicates the version of [CSI spec](https://github.com/container-storage-interface/spec) that Cinder CSI supports whereas `Version` is the version of Cinder CSI driver itself.

Reply:
> If the CSI spec version is upgraded, the Cinder CSI version need bump as well.

Would you please provide an example?

Reply:
In which case should `y` be upgraded?